ch04.adoc

```diff
@@ -204,310 +220,236 @@ Optionally, a time coordinate variable may be indicated additionally by providin
 
-* _Z_ is the time zone offset.
-This is an interval of time, specified in one of the formats described below.
-Only numbers (digits, `+`, `-` and `:`) or the letter "Z" are allowed in _Z_, not time zone names or acronyms.
+* __S__ is second, which may be integer or floating point (see <<leap-seconds>>),
 
-The default time zone offset is zero.
-In a time zone with zero offset, time (approximately) equals mean solar time for 0 `degrees_east` of longitude.
-(Although this may be exact in a model, in reality the time with zero time zone offset differs by some seconds from mean solar time; see the discussion of UTC and leap seconds in <<calendar>>.)
-If both time and time zone offset are omitted the time is 00:00:00 (the beginning of the day i.e. midnight at 0 `degrees_east`).
-Thus, **`units = "days since 1990-1-1"`** means the same as **`units = "days since 1990-1-1 00:00:00"`**.
+* __T__ is the time zone offset.
+__T__ is __not__ a time zone name or acronym; it is an interval of time.
 
-The time zone offset _Z_ must be in one of the following five formats, where numeric hours may optionally be prefixed with a `+` or `-` sign:
+The default for time zone offset __T__ is zero, which may also be explicitly indicated in any of the numeric formats for __T__ defined below, or by the letter **`Z`**, sometimes referred to as "Zulu Time".
+We suggest that a zero offset be stated explicitly to avoid confusion in situations where omitting it might be misunderstood as indicating local time.
+We recommend that a non-zero time zone offset should __not__ be specified __in any situation__, because it is easy to make mistakes about the sign of the offset and allowance for daylight-saving/summer time.
+A non-zero offset is __not allowed__ in the **`utc`** and **`tai`** calendars (see <<calendar>>).
 
-** The letter `Z` indicating zero offset, sometimes referred to as "Zulu Time".
+In a time zone with zero offset, time (approximately) equals mean solar time for 0 **`degrees_east`** of longitude.
+If both time and time zone offset are omitted the time is 00:00:00 (the beginning of the day i.e. midnight in the time zone with zero offset).
+Thus, **`units = "days since 1990-1-1"`** means the same as **`units = "days since 1990-1-1 00:00:00Z"`**.
 
-** _H_, the hour alone, of one or two digits e.g. **`-6`**, **`2`**, **`+11`**, which is sufficient for many time zones.
+Other than **`Z`** for zero offset, the time zone offset __T__ must be in one of the following four signed numeric formats, where &pm; stands for the positive sign **`+`** or the negative sign **`-`**, and __n__ stands for a single digit:
 
-** __H__:__M__, where _H_ is hour and _M_ minute, each of one or two digits, e.g. **`5:30`**.
+* &pm;__H__, the hour alone, of one or two digits e.g. **`-6`**, **`-0`**, **`+0`**, **`+11`**, which is sufficient for many time zones.
```

Comment:
```suggestion
* &pm;__H__, the hour alone, of one or two digits e.g. **`-6`**,**`+0`**, **`+11`**, which is sufficient for many time zones.```

We must not allow `-0` or `-00`!!  It is not included  in widely used and generally accepted standards (ISO), and in some it has a special meaning (something like "it's UTC time with unknow timezone"). Currently UDUNITS2 accepts this, which must not be used as an argument because its datetime handling is pretty fragile and buggy (I am currently doing some work on this).